### PR TITLE
docs: update website to Biome

### PR DESCRIPTION
**Pull request by Conaclos**
## Summary

This PR updates the website by renaming Rome to Biome and updating links.

## Test Plan

CI.

### CODE REVIEW

File: editors/vscode/README.md [hunk 1]
```diff
@@ -27,13 +27,13 @@ Configure Rome as the default formatter for supported files to ensure that VS Co
 You can also enable Rome for specific languages only:
 
 - [Open the `settings.json`](https://code.visualstudio.com/docs/getstarted/settings#_settingsjson): open the _Command Palette_(<kbd>Ctrl</kbd>/<kbd title="Cmd">⌘</kbd>+<kbd title="Shift">⇧</kbd>+<kbd>P</kbd>) and select _Preferences: Open User Settings (JSON)_
-- And set the `editor.defaultFormatter` to `biome.rome` for the desired language
+- And set the `editor.defaultFormatter` to `biomejs.biome` for the desired language
```
Comment (ematipico) [inline editors/vscode/README.md:30]: Is this correct?

**Review by ematipico — COMMENTED** [review]
We're doing the same job 😅

File: editors/vscode/README.md (hunk 1 above)
Comment (Conaclos) [reply editors/vscode/README.md:30]: The publisher is named `biomejs`. We could publish a package named `biome`?

Comment (ematipico) [reply editors/vscode/README.md:30]: Oh no, I am entirely on board with it! Consistency FTW

**Pull request merged**